src/components/SaveExport/ExportDialog.jsx

```diff
@@ -94,6 +103,71 @@ export default () => {
         bitsSeparatedToggled,
     ];
 
+    const setExportIndexes = (begin, end) => {
+        setIndexBegin(begin);
+        setIndexEnd(end);
+    };
+
+    const [radioValue, setRadioValue] = useState(0);
+    const radios = [
```

Comment:
Maybe a more specific name for this? E.g. `exportSelections` or something like that?